_sass/_footer.scss

```diff
@@ -1,8 +1,10 @@
 // Site footer
 
 .site-footer {
-  margin: 3em 0;
```

Comment:
I think we might want more spacing on top and bottom if we have a different background. I'm looking through the secondary pages (e.g., events) and it the spacing feels off
![screen shot 2017-09-11 at 12 18 53 pm](https://user-images.githubusercontent.com/4692834/30285214-69fc2108-96eb-11e7-8719-2227bcb6593d.png)

Reply:
good idea, it was looking wierd with the margin, but padding looks better.

Reply:
Minor adjustments in 029191b